shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -1871,7 +1876,6 @@ - (void)performOrientationUpdate:(UIInterfaceOrientationMask)new_preferences {
         if (!windowScene) {
           // When the view is not loaded, it does not make sense to access the interface
           // orientation, bail.
-          FML_LOG(WARNING) << "Trying to access the window scene before the view is loaded.";
```

Comment:
is this intended change?

Reply:
The log is a bit redundant, since the windowSceneIfViewLoaded function that is called in the line above already logs the message.

Reply:
Although, I guess there could be a case where the windowScene could be null for other reasons. Maybe rather than delete the line, I should log out the comment instead?

Reply:
yeah even if both are logged, one is for window scene, and the other is for screen, so not exactly duplicate.